File: Lab6/WebApplicationTestTask.Models/Order/OrderPresentationModel.cs
using System;
using System.Collections.Generic;
using System.Text;
using WebApplicationTestTask.Entities.Enums;
using WebApplicationTestTask.Models.OrderProduct;

namespace WebApplicationTestTask.Models.Order
{
    public class OrderPresentationModel
    {
        public int Id { get; set; }
        public int CustomerId { get; set; }
        public string CustomerName { get; set; }
        public string CustomerAddress { get; set; }
        public decimal TotalCost { get; set; }
        public OrderStatus OrderStatus { get; set; }
        public string Comment { get; set; }
        public DateTime OrderDate { get; set; }
        public List<OrderProductPresentationModel> OrderProducts { get; set; }
    }
}

File: Lab6/WebApplicationTestTask/client-app/src/app/orders/models/order-presentation-model.ts
import { OrderStatus } from "../enums/order-status";
import { OrderProductPresentationModel } from "./order-product-presentation-model";

export class OrderPresentationModel {
    id: number;
    customerId: number;
    customerName: string;
    customerAddress: string;
    totalCost: number;
    orderStatus: OrderStatus
    orderProducts: OrderProductPresentationModel[];
    orderDate: number;
    comment: string;
}